pkg/mutate/image.go

```diff
@@ -151,6 +153,10 @@ func (img *image) populate() error {
 		manifest.Config.Data = config
 	}
 
+	if configArtifactType != "" && configType == "application/vnd.oci.empty.v1+json" {
```

Comment:
What's the rationale for checking `configType == "application/vnd.oci.empty.v1+json"` here? I don't think the spec mandates that ... and it looks like the GGCR code sets it whenever an unrecognized config media type is found (https://github.com/tri-adam/go-containerregistry/blob/062c74e0d745d850e89a62512c32edd924ce9310/pkg/v1/remote/fetcher.go#L168-L170)?